packages/fxa-content-server/server/lib/beta-settings.js

```diff
@@ -74,6 +74,7 @@ const settingsConfig = {
   appleAuthConfig: config.get('appleAuthConfig'),
   brandMessagingMode: config.get('brandMessagingMode'),
   glean: { ...config.get('glean'), appDisplayVersion: config.get('version') },
+  redirectAllowList: config.get('redirect_check.allow_list'),
```

Comment:
Just realized I must be missing a step to make this populate, I'm only seeing the default value (`['localhost']`) for this; will sort out tomorrow.